x/bank/client/rest/sendtx.go

```diff
@@ -8,16 +8,15 @@ import (
 	"github.com/cosmos/cosmos-sdk/client/context"
 	clientrest "github.com/cosmos/cosmos-sdk/client/rest"
 	"github.com/cosmos/cosmos-sdk/codec"
-	"github.com/cosmos/cosmos-sdk/crypto/keys"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/rest"
 
-	"github.com/cosmos/cosmos-sdk/x/bank"
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
 )
 
 // RegisterRoutes - Central function to define routes that get registered by the main application
-func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec, kb keys.Keybase) {
-	r.HandleFunc("/bank/accounts/{address}/transfers", SendRequestHandlerFn(cdc, kb, cliCtx)).Methods("POST")
+func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec) {
+	r.HandleFunc("/bank/accounts/{address}/transfers", SendRequestHandlerFn(cdc, cliCtx)).Methods("POST")
```

Comment:
This is another argument for moving the `send` command into the bank module.

Reply:
I'm confused what is the argument? - this code change just removes extra dead variables

Reply:
The fact that we have the rest route in bank, but the send command is elsewhere. That is not in line with user expectations.

Reply:
where does the send command live rn ?

Reply:
Que? The send command lives in bank @jackzampolin. I think you mean the `/bank/balances/{address}` endpoint needs to move from auth to bank ;-)

Reply:
No, I'm talking about the following CLI commands that are currently not under a module:

```
tx:
  send        Create and sign a send tx
              
  sign        Sign transactions generated offline
  multisign   Generate multisig signatures for transactions generated offline
  broadcast   Broadcast transactions generated offline
  encode      Encode transactions generated offline

query:
  tendermint-validator-set Get the full tendermint validator set at given height
  block                    Get verified data for a the block at given height
  txs                      Search for paginated transactions that match a set of tags
  tx                       Find a transaction by hash in a committed block.
              
  account                  Query account balance
```

Reply:
@rigelrozanski anything to further do or discuss here? Should further discussion move elsewhere?

Reply:
As discussed in the call, I've added module to the CLI (bank/auth) for `tx` and `query`.... in addition the existing "top-level-within-tx/query" commands from bank/auth remain where they are as well as aliases